_optimisations/r/vectorisation.md

```diff
@@ -81,6 +81,11 @@ Using vectorisation is both more concise and more performant.
 
 You might be tempted to use `sapply()` or `vapply()`, which allow you to apply a transformation function to each element within an array, but these were slower than the original loop in testing.
 
+This is because the `apply()` family of functions, in contrast to popular believe, are _not_ more performant than `for` loops in R.
+In fact, the `apply()` implementation contains a regular R `for` loop. The reason they may sometimes be more performant than `for` loops
+is because they take care of the common pitfalls with loops, like the [growing data problem](/optimisation/r/dont-grow-dataframes).
```

Comment:
```suggestion
This is because the `apply` family of functions, in contrast to popular believe, are _not_ more performant than `for` loops in R.
In fact, the `apply` implementation contains a regular R `for` loop. The reason they may sometimes be more performant than `for` loops
is because they take care of the common pitfalls with loops, like the [growing dataframes problem](/optimisation/r/dont-grow-dataframes).
```

Given `apply` refers to the family, rather than a particular method better to match with your final line imo.